lib/toolkit/loader.js

```diff
@@ -431,9 +444,14 @@ const nodeResolve = iced(function nodeResolve(id, requirer, { rootURI }) {
   if ((resolvedPath = loadAsDirectory(fullId)))
     return stripBase(rootURI, resolvedPath);
 
+  // If the requirer is an absolute URI then the node module resolution below
+  // won't work correctly as we prefix everything with rootURI
+  if (isAbsoluteURI(requirer))
+    return void 0;
```

Comment:
It doesn't appear that we have any tests for this use case in test-native-loader.js, we should add some since I'm assuming we are using the use case somehow?